README.md

```diff
@@ -28,7 +28,7 @@ module "eh_namespace" {
 
 | Name | Version |
 |------|---------|
-| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | >= 3.58.0 |
+| <a name="provider_azurerm"></a> [azurerm](#provider\_azurerm) | 3.59.0 |
```

Comment:
This change happens when you re-generate the docs after a `terraform init` but really it should remain as `>= 3.58.0` which is what is specified in the provider.